addons/binding/org.openhab.binding.chromecast/ESH-INF/thing/thing-types.xml

```diff
@@ -106,10 +236,209 @@
+        <state readOnly="true"/>
+    </channel-type>
+
+    <channel-type id="albumArtist" advanced="true">
+        <item-type>String</item-type>
+        <label>Album Artist</label>
+        <description>The name of the album's artist</description>
+        <state readOnly="true"/>
+    </channel-type>
+
+    <channel-type id="albumName" advanced="true">
+        <item-type>String</item-type>
+        <label>Album Name</label>
+        <description>The name of the album</description>
+        <state readOnly="true"/>
+    </channel-type>
+
+    <channel-type id="artist" advanced="true">
+        <item-type>String</item-type>
+        <label>Artist</label>
+        <description>The name of the artist</description>
+        <state readOnly="true"/>
     </channel-type>
 
+    <channel-type id="broadcastDate" advanced="true">
+        <item-type>Number</item-type>
+        <label>Broadcast Date</label>
+        <description>The broadcast date of the currently playing media</description>
+        <state readOnly="true"/>
+    </channel-type>
+
+    <channel-type id="composer" advanced="true">
+        <item-type>String</item-type>
+        <label>Composer</label>
+        <description>The composer of the current track</description>
+        <state readOnly="true"/>
+    </channel-type>
+
+    <channel-type id="creationDate" advanced="true">
+        <item-type>Number</item-type>
+        <label>Creation Date</label>
+        <description>The creation date of the currently playing media</description>
+        <state readOnly="true"/>
+    </channel-type>
+
+    <channel-type id="discNumber" advanced="true">
+        <item-type>Number</item-type>
+        <label>Disc Number</label>
+        <description>The disc number of the currently playing media</description>
+        <state readOnly="true"/>
+    </channel-type>
+
+    <channel-type id="episodeNumber" advanced="true">
+        <item-type>Number</item-type>
+        <label>Episode Number</label>
+        <description>The episode number of the currently playing media</description>
+        <state readOnly="true"/>
+    </channel-type>
+
+    <channel-type id="height" advanced="true">
+        <item-type>Number</item-type>
+        <label>Height</label>
+        <description>The height of the currently photo</description>
+        <state readOnly="true"/>
+    </channel-type>
+
+    <channel-type id="images" advanced="true">
+        <item-type>String</item-type>
+        <label>Images</label>
+        <description>The available images as a json array</description>
+        <state readOnly="true"/>
+    </channel-type>
+
+    <channel-type id="locationName" advanced="true">
+        <item-type>String</item-type>
+        <label>Location Name</label>
+        <description>The location of where the current media was taken</description>
+        <state readOnly="true"/>
+    </channel-type>
+
+    <channel-type id="locationLatitude" advanced="true">
```

Comment:
It would make a lot of sense if you could combine `locationLatitude` and `locationLongitude` into a `location` channel, which could then have `Location` item type.